docs/changelog.md

```diff
@@ -235,30 +261,41 @@ description: |
     arithmetic is needed, such as for finance
   - Added `std` module for accessing standard library definitions even when a
     variable with the same name shadows/overwrites it
+    <!-- I can't find this module in the search. It should have a docs page. -->
   - Added [`array.to-dict`], [`array.reduce`], [`array.windows`] methods
   - Added `exact` argument to [`array.zip`]
   - Added [`arguments.at`] method
   - Added [`int.from-bytes`], [`int.to-bytes`], [`float.from-bytes`], and
     [`float.to-bytes`]
-  - [`calc.round`] no longer accepts negative digits (**Minor breaking change**)
+  - `digits` parameter of [`calc.round`] no longer accepts negative integers
+    (**Minor breaking change**)
   - Conversions from [`int`] to [`float`] will now error instead of saturating
     if the float is too large (**Minor breaking change**)
+    <!-- Pretty sure docs should be updated here:
+         https://staging.typst.app/docs/reference/foundations/float/#constructor
+    -->
   - Added `float.nan` and `float.inf`, removed `calc.nan`
     (**Minor breaking change**)
+    <!-- Docs missing a simple example of printing and/or making these values. -->
```

Comment:
don't know what you mean here

Reply:
There is https://staging.typst.app/docs/reference/foundations/float/#definitions-is-nan, but there is not a single instance of something like this:

```typ
#let nan = <create NaN>
#nan
#let infinity = <create infinity>
#infinity
```

There should be an example of how the values can generally be constructed and how would they print out. If direct value isn't pretty, maybe `repr()` can be used for better output. I haven't tried doing any of that.

Reply:
I'm confused. It literally says above that `float.nan` was added. `#let nan = float.nan`.